### ISSUE TITLE
Tox-UI doesn't have a single image, that shows all available graphic options.

### ISSUE THREAD

**Issue by GrayHatter** [ItsDuke/Tox-UI#11]
Something similar to this http://th07.deviantart.net/fs70/PRE/i/2011/047/5/2/128x128_black_icon_pack_by_innocentlow-d39nx0q.jpg

Just a quick way to see what a graphic should look like, or what options a dev might want to use.

**Comment by ItsDuke** [comment]
@GrayHatter graphic...options? Guess I'm out of the loop on what you mean on that. From the image you provide it looks like you mean a page that's got every icon on it, is that what you meant?

**Comment by GrayHatter** [comment]
yeah, I was looking through the images on github, but everything is small and white with a white/gray alpha background. Making it impossible to actually see the images

**Comment by ItsDuke** [comment]
Does the default image here: https://github.com/ItsDuke/Tox-UI not do a good enough job of showing what typical UI items look like?
